mscore/musescore.cpp

```diff
@@ -1406,7 +1406,7 @@ void MuseScore::setCurrentScoreView(ScoreView* view)
             synthControl->setScore(cs);
       if (selectionWindow)
             selectionWindow->setScore(cs);
-      if (mixer)
+      if (mixer && cs)
```

Comment:
Perhaps better to still do the updateAll(), but do it on cs itself if null.  That is,

if (mixer)
      mixer->updateAll(cs : cs->masterScore() : nullptr);

Not that I've tried it.  But without the update I might worry somehow the mixer still has a reference to the old closed score and might crash at some point in the future.